contact-profile/contact-profile-project/contact-profile-app/components/modules/ContactWithAssociations/fields.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ModuleFields, ColorField } from '@hubspot/cms-components/fields';
+
+export const fields = (
+  <ModuleFields>
+    <ColorField
+      label="Button color"
+      name="button_color"
+      default={{ color: '#F7761F' }}
+    />
```

Comment:
Hm... do things error if `<ModuleFields>` is empty? (I assume that is why you'd leave an unused color field here?)

Reply:
They do - I think we should probably allow exporting null, or just handle that module export being undefined.